documentation/modules/ROOT/pages/connectors/oracle.adoc

```diff
@@ -4183,6 +4187,8 @@ If the captured table(s) schema changes infrequently or never, this is the ideal
 `hybrid`:: Uses a combination of the database's current data dictionary and the {prodname} in-memory schema model to resolve table and column names seamlessly.
 This mode performs at the level of the `online_catalog` LogMiner strategy with the schema tracking resilience of the `redo_log_catalog` strategy while not incurring the overhead of archive log generation and performance costs of the `redo_log_catalog` strategy.
 
+`dictionary_from_file`:: Uses dictionary from file located in a Oracle directory object. Use this option if you set the log.mining.read.only to true and informed a log.mining.readonly.hostname.
```

Comment:
I believe we're also missing the `log.mining.path.dictionary` configuration property, and I suspect where you mention "from file located in a Oracle directory object" should likely refer to the path configured in the aforementioned property. We do this using xrefs in a few other properties, follow those as guides.

I would also suggest two other changes

1. Add a section under the "Log Mining Strategies" that outlines the `dictionary_from_file` option. 
2. Add a new section under "How the connector works" that outlines how one would setup capturing changes from a read-only standby using the `dictionary_from_file` strategy.

What do you think?